src/ESPAsyncWebServer.h

```diff
@@ -128,6 +128,7 @@ class AsyncWebServerRequest {
     AsyncWebHandler* _handler;
     AsyncWebServerResponse* _response;
     StringArray _interestingHeaders;
+    std::function<void(void)> _onDisconnectfn;
```

Comment:
why not typedef this like it is done in AsyncTCP?

Reply:
You means doing ?
`typedef std::function<void(void)> ArCallbackHandler;`

Because I was just lazy to add extra line ^_^ - but I can do it if you want :+1: 
Just confirm the name you want for the typedef :cat:

Reply:
maybe ArDisconnectHandler?

Reply:
Sure will do that -  thank you